pkgs/top-level/python-packages.nix

```diff
@@ -782,6 +782,8 @@ in {
 
   python-mnist = callPackage ../development/python-modules/python-mnist { };
 
+  pythonocc-core = callPackage ../development/python-modules/pythonocc-core { };
```

Comment:
`toPythonModule` needs to be used here so it can be used in `python.buildEnv`.

Reply:
Thanks! Done.